pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -718,23 +380,106 @@ PageServer.prototype = {
                         console.log(ex);
                     })
                     .always(function() {
-                        link.toggleClass("hidden", !version);
-                        link.prev().toggleClass("hidden", !version);
-                        link.text(version);
+                        self.setState({ sysroot_version_link_visible: version });
+                        self.setState({ sysroot_version_link_text: version });
```

Comment:
Oh, so *this* is the ominous "Version:" thing that I see on the "Configuration" card. It's not coming from /etc/os-release, but ostreed. I don't actually see this on master, the front page for me says "Operating System
Fedora 31 (Thirty One)" and no single "31" link anywhere. So maybe we should just kick that out and leave the ostree management to cockpit-ostree and its "Software Updates" page? It seems to be pretty dead on master already anyway.

If you want to keep it: This doesn't need two state variables for "visible" and "text", as they are set together, and to the same value.

Reply:
Removed.